src/js/components/Breadcrumb.js

```diff
@@ -183,56 +215,14 @@ class Breadcrumb extends React.Component {
 
     return expandedWidth >= availableWidth + COLLAPSE_BUFFER;
   }
-  /**
-   * Builds a crumb or multiple crumbs from the label and matching route
-   * @param  {String} label Url path part
-   * @param  {String} route Matching route name
-   * @param {Object} routeParams Router routeParams Object
-   * @param {Bool} labelIsParam Boolean indicating if the label represents a
-   * route parameter.
-   * @return {Array} Array of crumbs containing the label, route and route
-   * params. The returned label is the text shown for the crumb. The route is
-   * the route name which will be set on the Link-to attribute. The params will
-   * also be set on the Link tag. Each returned crumb will be spliced into the
-   * resulting breadcrumb.
-   */
-  buildCrumb(label, route, routeParams, labelIsParam) {
-    // Split label on whitespace and -
-    let splitLabelRegex = /[\s-]+/;
-    let joinLabelCharacter = ' ';
-    let params = {};
-
-    if (labelIsParam) {
-      // Extract param value as label e.g. (:userID -> Foo)
-      // and add param to params.
-      //
-      // Remove colon first
-      let paramName = label.slice(1);
-      // Decode value
-      label = decodeURIComponent(routeParams[paramName]);
-      params[paramName] = label;
-    } else {
-      // Split Label on space and - then capitalize each word and join.
-      // /foo-bar/ --> Foo Bar
-      label = label.split(splitLabelRegex).map(function (word) {
-        return StringUtil.capitalize(word);
-      }).join(joinLabelCharacter);
-    }
-    // Can return multiple route objects to get rendered in crumbs.
-    // Useful for splitting a route parameter like a file path.
-    return [{label, route, params}];
-  }
 
   renderCrumbsFromRoute() {
-    let {buildCrumb, shift} = this.props;
-
-    if (buildCrumb == null) {
-      buildCrumb = this.buildCrumb;
-    }
+    let {buildCrumb, buildCrumbCustomParams, shift} = this.props;
```

Comment:
won't the `buildCrumb` at the top of the file never be used because of this right here?